README.md

```diff
@@ -40,7 +40,7 @@ Options:
   --dev                                                    Flag to turn on the dev mode.  false by default.
   --dev.block-interval <Int32>                             The time interval between blocks. It's unit is milliseconds. Works only when dev mode is on.  10000 (ms) by default. (Default: 10000)
   --dev.reorg-interval <Int32>                             The size of reorg interval. Works only when dev mode is on.  0 by default. (Default: 0)
-  --log-minimum-level <String>                             The log minimum level during standalone execution. (Default: debug)
+  --log-minimum-level <String>                             The log minimum level during headless execution. (Default: debug)
```

Comment:
```suggestion
  --log-minimum-level <String>                             The log minimum level during headless execution.  debug by default. (Default: debug)
```
This line does not match with https://github.com/planetarium/NineChronicles.Headless/blob/7bbf8d99b682ec9cf8ca39518022f43f390cf2e2/NineChronicles.Headless.Executable/Program.cs#L112